doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -3475,7 +3475,7 @@ \subsubsection{Inputs}\label{inputs-9-014}
 
 \paragraph{Field: Cooling Capacity Function of Temperature Curve Name}\label{field-cooling-capacity-function-of-temperature-curve-name-3}
 
-The CoolCapfT curve represents the fraction of the cooling capacity of the chiller as it varies with temperature. The curve is normalized so that at design conditions the value of the curve should be 1.0. The curve is usually a biquadratic or bilinear curve with the input variables being the leaving chilled water temperature and the entering condenser water temperature (see \emph{Temperature Curve Input Variable} below). The available cooling capacity is computed as follows:
+The CoolCapfT curve represents the fraction of the cooling capacity of the chiller as it varies with temperature. The curve is normalized so that at design conditions the value of the curve should be 1.0. The curve is usually a biquadratic or bilinear curve with the input variables being the leaving chilled water temperature and the entering condenser water temperature (see \emph{Temperature Curve Input Variable} below). If the chiller is AirCooled, the temperature of the outdoor air node is used for the condenser temperature. The available cooling capacity is computed as follows:
```

Comment:
Same comment as above.  ". . . the temperature of the condenser inlet node (outdoor air node) is used . . ."